xmlserializer/XmlElement.cpp

```diff
@@ -271,3 +262,18 @@ bool CXmlElement::CChildIterator::next(CXmlElement& xmlChildElement)
     return false;
 }
 
+template bool CXmlElement::getAttribute(const std::string &name, std::string &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint16_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, int32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint64_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, bool &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, double &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, float &value) const;
+
+template void CXmlElement::setAttribute(const std::string& name, const std::string &value);
+template void CXmlElement::setAttribute(const std::string& name, const bool &value);
+template void CXmlElement::setAttribute(const std::string& name, const int32_t &value);
+template void CXmlElement::setAttribute(const std::string& name, const uint32_t &value);
+template void CXmlElement::setAttribute(const std::string& name, const uint64_t &value);
+template void CXmlElement::setAttribute(const std::string& name, const float &value);
```

Comment:
Why is there not a many as previously ?

Maybe you could do that in a loop. (that was a joke)